homework/shelkovich_ilya/arrayService/src/test/java/com/tutrit/java/quickstart/service/ArrayServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ArrayServiceTest {
+
+    ArrayService arrayService;
+
+    @Before
+    public void setUp() {
+        this.arrayService = new ArrayService();
+    }
+
+    @Test
+    public void createArray() {
+        Assert.assertArrayEquals(new int[]{}, arrayService.createArray(0));
+    }
+
+    // не уверен что этот тест проверяет то, что я хотел. Я хотел проверить заполняется ли массив случайными числами
+    @Test
+    public void fillRandomly() {
+        int[] array = new int[10];
+        Assert.assertArrayEquals(array, arrayService.fillRandomly(array));
+    }
+
+    @Test
+    public void printArray() {
+        int[] array = {10, 12, 14};
+        arrayService.printArray(array);
+        Assert.assertTrue(true);
```

Comment:
ptal, seriously )

Reply:
you could just print, that fine for this test.

Reply:
fixed it